### PR TITLE
Chore/better organisation

### PR DESCRIPTION
**Pull request by Sefiraat**

### CODE REVIEW

**Comment by Sefiraat** [comment]
Don't merge just yet :)

**Review by J3fftw1 — CHANGES_REQUESTED** [review]
After this LGTM

File: src/main/java/dev/sefiraat/netheopoiesis/api/events/CruxSpreadEvent.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    public NetherSeed getGrowingPlant() {
+        return growingPlant;
+    }
+
+    /**
+     * @return The crux type that will be spread
+     */
+    @Nonnull
+    public NetherCrux getGrowthStage() {
+        return crux;
```
[the remark this answers is not in the document]
Comment (Sefiraat) [reply src/main/java/dev/sefiraat/netheopoiesis/api/events/CruxSpreadEvent.java:75]: That's not nit picky, its just plain wrong on my part, ha

**Comment by J3fftw1** [comment]
LGTM.

**Review by J3fftw1 — APPROVED** [review]

**Pull request merged**